### DISCUSSION TITLE
Can I use any bootstrap 5 template for a quarto website?

### DISCUSSION THREAD

**Discussion by durraniu** [quarto-dev/quarto-cli#9079] [Q&A] [answered]
### Description

I used a random bootstrap 5 template which contains various CSS files. After adding them in the CSS option in quarto yml file, the website seemed to work. But the navigation bar theme and margins are not overridden by the new theme. How do I completely override quarto website theme/CSS?

**Comment by mcanouil** [comment]
All is described in <https://quarto.org/docs/output-formats/html-themes.html>.
Theme has to be passed under `theme` as SCSS not as CSS.

**Reply by durraniu** [reply]
Does that mean CSS files won't work if I don't have scss? The template I am trying to use does not have any scss files.

**Reply by mcanouil** [reply, marked as answer]
You have a Bootstrap theme that does not use variables?

Anyhow, you can put your CSS in a SCSS file as long as you follow the documentation I linked.

**Reply by durraniu** [reply]
Thank you.
